modules/mono/editor/GodotSharpTools/Project/ProjectGenerator.cs

```diff
@@ -170,6 +170,7 @@ public static ProjectRootElement CreateLibraryProject(string name, out ProjectPr
             releaseGroup.Condition = " '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ";
             releaseGroup.AddProperty("DebugType", "portable");
             releaseGroup.AddProperty("Optimize", "true");
+            releaseGroup.AddProperty("DefineConstants", "$(Constants);GODOT;");
```

Comment:
Can we name it `GodotDefineConstants` instead of `Constants`?

Reply:
Sure